### PR TITLE
Add deployment file for the 2nd test case and update README.md

### PR DESCRIPTION
**Pull request by daisy-ycguo**

### CODE REVIEW

File: tests/testcases/triggerrule/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+application:
+  name: wskdeploy-samples
+  namespace: /wiskdeploy/samples/
```
Comment (paulcastro) [inline tests/testcases/triggerrule/deployment.yml]: should we say `/wskdeploy/samples` ?

**Comment by mrutkows** [comment]
Both the manifest and deployment files look good.  However, we recently renamed "location" key under the Action entity to be "function".   This was noted on a recent call, but perhaps went un-noticed.  The goal was to align the name for the functional code artifact (semantics) to other Serverless offerings, but perhaps we should change back to "location" for now that way all entities can have the same keyword.  Let us leave "location" for now and discuss on next specification call with Paul.

**Comment by mrutkows** [comment]
Please note that the project was renamed wskdeploy and your branch is wsktool still

**Comment by daisy-ycguo** [comment]
So I will keep "location" for now, till we decided to change to "function".

**Pull request merged**